nav2_bringup/launch/bringup_launch.py

```diff
@@ -108,6 +108,15 @@ def generate_launch_description():
             condition=IfCondition(use_namespace),
             namespace=namespace),
 
+        Node(
+            condition=IfCondition(use_composition),
+            name='nav2_container',
+            package='rclcpp_components',
+            executable='component_container_mt',
```

Comment:
I thought we didn't want to use the mt container and instead use the isolated one you added?

Reply:
yes, but `component_container_isolated` isn't available from binary now, maybe we can wait for next Rolling sync.